bokeh/server/tornado.py

```diff
@@ -120,6 +145,7 @@ def __init__(self, applications, prefix, hosts,
             raise ValueError("stats_log_frequency_milliseconds must be > 0")
 
         self._hosts = set(hosts)
+        self._hosts.add('127.*.*.*')
```

Comment:
Happy to remove though, my application goal is just to be able to opt-out, which the other changes allow.